packages/scaffolding-cli/templates/src/ssr/server/index.ts

```diff
@@ -46,7 +52,44 @@ export default app
     .prepare()
     .then(() => {
         const server = express()
-        // TODO: Enable passport when attempting https://amido-dev.visualstudio.com/Amido-Stacks/_sprints/taskboard/Cycle%203%20-%20Front%20End%20-%20Part%202/Amido-Stacks/Cycle%203%20-%20Front%20End%20-%20Part%202/Sprint%205?workitem=1800
+        if (authenticationEnabled) {
+            const sessionConfig = setPassportSessionCookie(isRedisEnabled, conf)
```

Comment:
Can this feature be split out into another file?

Reply:
Can you justify this please? The main purpose of Node.js here is to handle the auth and logging etc. It is pretty common to have auth related logic in server.ts.

Reply:
Totes get that, but if this is optional it might make it easier for our users to remove the unneeded code.

Reply:
Understood. Will make it optional in the next PR.